src/lib/darknode/utils/feesUtils.ts

```diff
@@ -39,13 +39,16 @@ export const getClaimFeeForToken = (symbol: string, blockState: BlockState) => {
     }
     const { gasLimit, gasCap, dustAmount } = data;
 
-    const fee = new BigNumber(gasLimit)
+    let fee = new BigNumber(gasLimit)
         .times(new BigNumber(gasCap))
         // RenVM subtracts `dustAmount + 1` to ensure the change back to itself
         // is greater than the dust amount.
         .plus(dustAmount)
         .plus(1);
-
+    const feeDivisor = AllTokenDetails.get(Token[symbol])?.feeDivisor;
+    if(feeDivisor) {
+        fee =  fee.div(new BigNumber(Math.pow(10, feeDivisor)));
```

Comment:
Rather than mixing BigNumber maths and floating point maths, this could instead be:
`fee = fee.shiftedBy(-feeDivisor);` or
`fee = fee.div(new BigNumber(10).exponentiatedBy(feeDivisor));`

Also, there seems to be an extra space after the `=`. You can format the files by running `yarn prettier -w './src/**/*.ts*'` before committing or enabling "Format on Save" if you are using an IDE.